qoqo_for_braket/qoqo_for_braket/backend.py

```diff
@@ -236,20 +256,34 @@ def _run_circuits_batch(
         """
         task_specifications: List[BraketCircuit] = []
         shots_list = []
-        readouts = []
+        metadata = []
         for circuit in circuits:
             (task_specification, shots, readout) = self._prepare_circuit_for_run(circuit)
+            (
+                output_bit_register_dict,
+                output_float_register_dict,
+                output_complex_register_dict,
+            ) = self._set_up_registers(circuit)
             task_specifications.append(task_specification)
             shots_list.append(shots)
-            readouts.append({"readout_name": readout})
+            metadata.append(
```

Comment:
Similarly here, we call this metadata but the docstrings still says `readout`

Reply:
Updated too